spec/wisper/activerecord/publisher_spec.rb

```diff
@@ -27,4 +31,70 @@ class User < ActiveRecord::Base
       user.destroy
     end
   end
+
+  context 'when specific broadcasts are specified' do
```

Comment:
I think there may be value in explicitly adding one more spec here.  the behavior i'd be looking to ensure is:
```
Given two models have been defined
When one model is configured with specific broadcasts
I expect the other model's broadcast events to _not_ be affected
```

The current implementation should fulfill this requirement, and this is more of an explicit test to ensure that the class_attribute does not change in the future to be reused at the Mixin level which would be undesirable behavior.

Reply:
@wireframe added!